packages/xrpl/src/Wallet/rfc1751.ts

```diff
@@ -176,15 +178,34 @@ function bufferToArray(buf: Buffer): number[] {
  * @param buf - A 128-bit (16 byte) buffer
  * @returns A buffer containing the same data with reversed endianness
  */
-function swap128(buf: Buffer): Buffer {
-  // Interprets buffer as an array of (two, in this case) 64-bit numbers and swaps byte order in-place.
-  const reversedBytes = buf.swap64()
-
-  // Swap the two 64-bit numbers since our buffer is 128 bits.
-  return Buffer.concat(
-    [reversedBytes.slice(8, 16), reversedBytes.slice(0, 8)],
-    16,
-  )
+function swap128(buf: Uint8Array): Uint8Array {
+  // // Interprets buffer as an array of (two, in this case) 64-bit numbers and swaps byte order in-place.
+  const reversedBytes = swap64(buf)
+  // // Swap the two 64-bit numbers since our buffer is 128 bits.
+  return concat([reversedBytes.slice(8, 16), reversedBytes.slice(0, 8)])
+}
+
+function swap(b: Uint8Array, n: number, m: number): void {
```

Comment:
nit: This function order is a bit confusing. I'd recommend ordering them in `swap`, `swap64`, `swap128` so that by the time you read the last one, you know all the details it depends on.

Reply:
(The opposite order is also fine - it's just odd to have a function in between two functions which both depend on it)